ansible/roles/run-test-target/tasks/test-collection-method.yml

```diff
@@ -65,6 +65,7 @@
   - name: Run integration tests
     become: "{{ runtime_as_root }}"
     shell: |
+      echo '/tmp/core.out' | sudo tee /proc/sys/kernel/core_pattern
```

Comment:
I wonder if the tests should continue to be responsible for setting the pattern. My thinking is that currently the name of the core-dump file is very ambiguous and difficult to match to a specific test. If we set the pattern before each test suite, when we start collector, to something like `/tmp/core-< test suite name >.dump` then we would be able to tie it to a specific failure.

Reply:
I will add the integration test names to the core dump file names in a different PR. In this PR I will just add timestamps to the core dump file names.

Reply:
That PR is here https://github.com/stackrox/collector/pull/1595